azkaban-common/src/test/java/azkaban/executor/InteractiveTestJob.java

```diff
@@ -137,6 +138,12 @@ public void succeedJob(final Props generatedProperties) {
     }
   }
 
+  public void ignoreCancel() {
+    synchronized (this) {
+      this.ignoreCancel = true;
```

Comment:
Actually the difference with the other flags is that when they're toggled there's also `this.notify();` so it's handy to cover both with the sync block..